ona/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
```

Comment:
I'm not sure you need serializers or views for this integration... the initial design was only that you would create submissions via the report creation. Not independently.